beets/dbcore/db.py

```diff
@@ -177,8 +248,8 @@ def __init__(self, db=None, **values):
         """
         self._db = db
         self._dirty = set()
-        self._values_fixed = {}
-        self._values_flex = {}
+        self._values_fixed = LazyConvertDict(self);
+        self._values_flex = LazyConvertDict(self);
```

Comment:
No semicolons necessary in Python. :smiley: :snake:

Reply:
Hah, force of habit ;-)